heron/executor/src/python/heron_executor.py

```diff
@@ -610,11 +610,20 @@ def _get_jvm_instance_options(self, instance_id, component_name, remote_debugger
             java_version.startswith("1.6") or \
             java_version.startswith("1.5"):
       java_metasize_param = 'PermSize'
+    xmn_param = '-Xmn%dM' % xmn_size
+    if self._get_java_major_version() >= 11:
+        # For Java 11 and above.
+        # The Xmx value is 25% of the available memory with a maximum of 25 GB.
```

Comment:
This information only applies to the Xmx default value. But it seems we are always setting Xmx on line 624. So perhaps this comment isn't needed? Or is it related to the Xmn change this PR introduces?

Reply:
Want to delete -Xmn.

If there is this setting, YoungHeap area is too limited, so if -Xmx4g, only 2g is used.

Reply:
Before deletion -Xmx=5.5g Used 2.25g
After deletion -Xmx=5.5g Used 4.9g

<img width="821" alt="image" src="https://user-images.githubusercontent.com/357785/158043683-e3952566-919a-422f-8a10-fdfc9cde2775.png">